src/components/nodes/utilities/shapeGroup.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Utility functions that operate on a collection of shapes
+ */
+
+export function getBoundingBox(shapes) {
```

Comment:
This is definitely replaceable by https://resources.jointjs.com/docs/jointjs/v3.1/geometry.html#g.Rect.prototype